modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -1049,9 +1043,22 @@ In {productname} 6.4.1, this issue has been resolved. Any pre-formatted elements
 === Editor increases the height indefinitely when using autoresize with content_css document
 //#TINY-8872
 
-In previous versions of {productname}, It was discovered that the editor had some issues with the autoresize function when using content_css documents, which caused the editor to have a indefinite height when content was being added.
+The `document` skin’s CSS aligns the bottom of the content area to the bottom of the editor, but does not set anything for `autoresize_bottom_margin`. As a consequence the {productname} editor takes the default value for this attribute: 50. Simultaneously, every time the `resize` function calculated the editor size it added this `autoresize_bottom_margin` value.
```

Comment:
```suggestion
The document skin’s CSS aligns the bottom of the content area to the bottom of the editor, but does not set anything for `autoresize_bottom_margin`. As a consequence the {productname} editor takes the default value for this attribute: 50. Simultaneously, every time the `resize` function calculated the editor size it added this `autoresize_bottom_margin` value.
```
_document_ is not an option or variable or other developer- or integrator-focussed term, so it shouldn’t be set in `monospace`.